resources/route53-resolver-endpoints.go

```diff
@@ -0,0 +1,90 @@
+package resources
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/route53resolver"
+	"github.com/rebuy-de/aws-nuke/pkg/types"
+)
+
+// Route53ResolverEndpoint is the resource type for nuking
+type Route53ResolverEndpoint struct {
+	svc  *route53resolver.Route53Resolver
+	id   *string
+	name *string
+	ips  []*route53resolver.IpAddressUpdate
+}
+
+func init() {
+	register("Route53ResolverEndpoints", ListRoute53ResolverEndpoints)
```

Comment:
```suggestion
	register("Route53ResolverEndpoint", ListRoute53ResolverEndpoints)
```

We always use the singular for a resource.